validators/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'info.solidsoft.pitest'
+
+dependencyCheck {
+    suppressionFile = "${rootDir}/config/owasp/suppressions.xml"
+}
+
+dependencies {
+    implementation project(':dao')
+    implementation project(':dao-api')
+    implementation project(':domain')
+    implementation project(':utils')
+    testImplementation project(":dao").sourceSets.unitTest.output
+    testImplementation project(":dao-api").sourceSets.unitTest.output
+    testImplementation project(":domain").sourceSets.unitTest.output
+    testImplementation project(":utils").sourceSets.unitTest.output
+
+    testImplementation project(":dao").sourceSets.integTest.output
+    testImplementation project(":dao-api").sourceSets.integTest.output
+    testImplementation project(":domain").sourceSets.integTest.output
+    testImplementation project(":utils").sourceSets.integTest.output
+}
+
+jar {
+    enabled = true
+    baseName = 'civil-sdt-domain'
```

Comment:
Is the baseName correct?  Going by other module build.gradle files I'd expect this to be "civil-sdt-validators"

Reply:
addressed

Reply:
Confirmed that baseName has been changed to civil-sdt-validators